app/templates/summary.html

```diff
@@ -37,7 +37,7 @@ <h1 class="saturn">{{content.summary.title}}</h1>
 
 {% include theme('partials/summary/summary.html') %}
 
-{% if content.is_view_submission_response_enabled %}
+{% if content.summary.is_view_submission_response_enabled %}
```

Comment:
Is this change related?

Reply:
No, I can move it into a separate PR or make a ticket?

Reply:
raise a ticket for it. then it doesn't muddy the waters